tests/datacon/neg/Data00.hs

```diff
@@ -1,4 +1,3 @@
-{-@ LIQUID "--expect-any-error" @-}
```

Comment:
I don't understand these removals. This is a neg test but it produces no expected errors?